frontend/packages/knative-plugin/integration-tests/features/managed-kafka connection/kafka-source-connection-to-kafka-connection-instance.feature

```diff
@@ -0,0 +1,59 @@
+Feature: Connection from Kafka Source to Managed Kafka Connection resource
```

Comment:
@sanketpathak dir name has space looks like `managed-kafka connection/kafka-source-connection-to-kafka-connection-instance.feature`

Reply:
and shouldn't this be in `knative-plugin` ?

Reply:
@invincibleJai this is inside the `knative-plugin` only

Reply:
ah my bad. still can fix the space thing

Reply:
Fixed